### ISSUE TITLE
Exited with code 2

### ISSUE THREAD

**Issue by Harold2017** [Microsoft/pyright#51]
**Describe the bug**
Use pyright in tox test env, the result has no errors but exited with code 2.
```bash
py36-pyright run-test: commands[0] | npx pyright -p .
Loading configuration file at /xxx/pyrightconfig.json
Finding source files
Found 8 source files
0 errors, 0 warnings 
Find Source Files:    0.002sec
Read Source Files:    0.001sec
Tokenize:             0.106sec
Parse:                0.264sec
Post-parse Walker:    0.215sec
Semantic Analyzer:    0.349sec
Type Analyzer:        1.339sec
ERROR: InvocationError for command /home/harold/.nvm/versions/node/v11.10.0/bin/npx pyright -p . (exited with code 2)

______________ summary ______________
ERROR:   py36-pyright: commands failed

```

**To Reproduce**
config json
```json
{
  "include": [
    "context",
    "tests"
  ],
  "ignore": [
    ".tox",
    ".pytest_cache"
  ],

  "typingsPath": "typestubs",
  "venv": "/xxx/virtualenvs/xxx-4EtOuWbY",

  "reportTypeshedErrors": true,
  "reportMissingImports": true,
  "reportMissingTypeStubs": false,

  "pythonVersion": "3.6",
  "pythonPlatform": "Linux",

  "executionEnvironments": [
    {
      "root": "testContext"
    }
  ]
}
```

**Expected behavior**
It indicates the reason if invocation with exit code 2 or not throw this error...

**command-line**
version: 1.0.8

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version (1.0.9).

**Comment by erictraut** [comment]
This is fixed in 1.0.9, which I just published. Thanks again for the report.

**Comment by Harold2017** [comment]
> This is fixed in 1.0.9, which I just published. Thanks again for the report.

Hi, Eric, unfortunately, i got a new error code now with your 1.0.9 version...
```bash
py36-pyright run-test: commands[0] | npx pyright -p .
Loading configuration file at /XXX/pyrightconfig.json
Searching for source files
Found 14 source files
0 errors, 0 warnings 
Completed in 2.981sec
ERROR: InvocationError for command /XXX/.nvm/versions/node/v11.10.0/bin/npx pyright -p . (exited with code 1)
```

**Comment by erictraut** [comment]
That's embarrassing. In fixing the original bug I introduced another one. It's now fixed, and I've done a better job this time confirming that it works. The fix will be in the next version (1.0.10).

**Comment by Harold2017** [comment]
Ha, it happens all the time :) 
Many thanks for your work!

**Comment by erictraut** [comment]
This is fixed in version 1.0.10, which I just published. If you still see the problem, please reopen the bug report.
